lib/api/graphQL.js

```diff
@@ -54,7 +54,7 @@ function buildGraphQL(options) {
 		/** @type {GraphQLResponse} */
 		const result = await makeRequest(`/graphql?${qs}`, { method: 'GET' });
 
-		return strict ? throwOnErrors(result) : result.data;
+		return strict || !result?.data ? throwOnErrors(result) : result.data;
```

Comment:
This feels a little odd to read. The actual behaviour is more like

```
if (!result.data) throwTheErrors()
return strict ? throwIfErrors(result) : result.data;
 ```
feels more like it captures the flow

Reply:
Perhaps we should explicitly check for undefined - an otherwise falsy value would indicate the server tried to return something.

```suggestion
		return strict || result?.data === undefined ? throwOnErrors(result) : result.data;
```

Reply:
Thanks. Done